CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.24](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.23...v4.0.0-alpha.24)
+
+_Apr 2, 2021_
+
+Big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇬🇷 Added elGR locale (#1275) @clytras
+- 🇪🇸 Added esES locale (#1286) @WiXSL
+- 🇯🇵 Added jaJP locale (#1283) @seed-of-apricot
+- 🇳🇱 Added nlNL locale (#1273) @wimdetroyer
+- 🐞 Bugfixes
+
+### @material-ui/x-grid@v4.0.0-alpha.24 / @material-ui/data-grid@v4.0.0-alpha.24
+
+#### Breaking Changes
+
+- [DataGrid] All slot components no longer get access to `GridBaseComponentProps` through the props. To use the `GridBaseComponentProps` call the `useGridSlotComponentProps` hook. (#1252) @DanailH
+- [DataGrid] Type `GridSlotsComponent` changed (#1252) @DanailH
+- [DataGrid] Rename `GridBaseComponentProps` type to `GridSlotComponentProps` (#1252) @DanailH
+- [DataGrid] Rename `useGridBaseComponentProps` hook to `useGridSlotComponentProps` (#1252) @DanailH
+- [DataGrid] Rename modules (#1292) @DanailH
+- [DataGrid] Renamed all events related to column reordering, e.g. `GRID_COL_REORDER_START` -> `GRID_COLUMN_REORDER_START` (#1299) @m4theushw
```

Comment:
Not sure if this should be mentioned in the CHANGELOG. In https://github.com/mui-org/material-ui-x/pull/1250 I proposed to rename them again.

Reply:
No preferences